libs/designsystem/header/src/header.component.html

```diff
@@ -0,0 +1,22 @@
+
+<div class="container" [ngClass]="{ centered: centered }">
```

Comment:
Consider deleting this additional container and apply the styles to the `:host` itself - and the centering to the host element instead using `@HostBinding` in header.component.ts file:
```
@HostBinding('class.centered')
@Input() centered = false;
```